packages/eth-rpc-adapter/src/__tests__/e2e/endpoint.test.ts

```diff
@@ -633,7 +633,7 @@ describe('endpoint', () => {
         blockNumber: '0xa',
         transactionIndex: '0x0',
         // gasPrice: '0x7b501b0da7',
-        gas: '0x1fd91',
+        gas: '0x25a7f',
```

Comment:
should use snapshot testing so it is just one command to update those values

Reply:
in this case it's a little tricky since we are testing `to.deep.contain()`, I didn't find something like `toMatchSnaphostshotContain()`

As a work around, I added `bail = 999` so one CI run should print out all outdated value, which should be pretty easy to manually change.

Reply:
check the snapshot test helpers I created https://github.com/AcalaNetwork/chopsticks/blob/master/packages/testing/src/check.ts
basically you can preprocess the object (e.g. filter to only keep the fields you want to check) and then assert it